src/libfetchers/git.cc

```diff
@@ -153,12 +159,13 @@ struct GitInputScheme : InputScheme
 
     std::pair<bool, std::string> getActualUrl(const Input & input) const
     {
-        // Don't clone file:// URIs (but otherwise treat them the
+        // file:// URIs are normally not cloned (but otherwise treat them the
         // same as remote URIs, i.e. don't use the working tree or
```

Comment:
```suggestion
        // file:// URIs are normally not cloned (but otherwise treated the
        // same as remote URIs, i.e. we don't use the working tree or
```

Reply:
Thanks, suggestion applied.